src/components/widget/ActionButton.js

```diff
@@ -240,6 +270,22 @@ class ActionButton extends Component {
         onBlur={this.handleDropdownBlur}
         onFocus={this.handleDropdownFocus}
       >
+        {promptOpen && (
+          <Prompt
+            title={promptTitle}
+            text={promptText}
+            buttons={{ submit: promptYes, cancel: promptNo }}
+            onCancelClick={() =>
+              this.setState({ promptOpen: false }, () => {
+                this.statusDropdown.blur();
+              })
+            }
+            onSubmitClick={() =>
+              this.processStatus(list.find((elem) => elem.key === 'VO'), true)
```

Comment:
why hardcoding?

Reply:
That sends to the processStatus only the obj with key VO . How else to send the void correctly ?